src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -818,8 +818,8 @@ public void composeTableReusedDatasets(XWPFTable xwpfTable){
                 }
 
                 if (reusedDatasets.get(i).getLicense() != null) {
-                    docVar.add(reusedDatasets.get(i).getLicense());
-                }
+                    //TODO second String license option for reused datasets.
+                    docVar.add("");                }
```

Comment:
What is missing for the implementation?

Reply:
this task is its own issue:
https://github.com/tuwien-csd/damap-backend/issues/89

the feature as a whole will take more effort than a small fix.
will require adapting the API, database and handling of the value internally (does not need to be sent to the frontend)

Reply:
Ah alright, thank you!